hyperbolic-rs/plot_embeds.py

```diff
@@ -235,5 +214,94 @@ def get_closest_points(src_embeds, dst_embeds, hyperbolic, curvature, top_k=15):
     return closest_indexes
 
 
+def load_model(ckpt_path, model_name, curvature, prep_data):
```

Comment:
could you say what the data structure in model_data is assumed to be?

Reply:
Done